### DISCUSSION TITLE
Only Wayland?

### DISCUSSION THREAD

**Discussion by Boyfinn** [SeaDve/Kooha#299] [Q&A] [answered]
Does this still support X11? Or is this a Wayland only application?
I'm still on X11 with Mint21.3 / Cinnamon 6.0.4. And even though Mint claims that the "Virginia" build is compatible with both Wayland and X11, I've only had trouble with Wayland so far, so I'm sticking with X11 for now.

**Comment by thepragmaticmero** [comment]
wayland > x11

**Comment by SeaDve** [comment, marked as answer]
As long as your distro implements screencast portal on X11, Kooha also does.

**Comment by e8hffff** [comment]
There's no point supporting X11 now with distros ditching X11. X11 has SimpleScreenRecord which is the best, but wont work on Wayland.
